src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background: url(../images/star.svg) no-repeat center;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+
+    &:nth-child(1),
+    &:nth-child(-n+2),
+    &:nth-child(-n+3),
+    &:nth-child(-n+4),
+    &:nth-child(-n+5) {
+      background-image: url("../images/star-active.svg");
+    }
```

Comment:
try to use cycle for
more detail in [video](https://mate.academy/learn/css-advanced/sass#/video/926)